includes/news/v2/mail/featured-stories.php

```diff
@@ -28,12 +28,12 @@
 									</td>
 								</tr>
 								<tr>
-									<td class="montserratbold" style="padding-left: 0; padding-right: 0; font-family: Helvetica, Arial, sans-serif; font-size: 24px; font-weight: bold; padding-top: 15px; padding-bottom: 4px; line-height: 1.1; color: #000; text-align: left;" align="left">
```

Comment:
Cells that use the `montserratsemibold` class will need to have `font-weight: 500` set on them explicitly to make sure browsers/clients display the font weight consistently (see Firefox; these headings are still a little heavy compared to Chrome)